flink-libraries/flink-table/pom.xml

```diff
@@ -92,6 +92,11 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.hive</groupId>
```

Comment:
Hi @twalthr . Thank you for your suggestions. I've moved all the code to hcatalog in commit 97968fb86af105f0fc8b68e8083998dd0725deef.